docs/src/man/man1/xhc-whb04b-6.1.adoc

```diff
@@ -88,22 +76,23 @@ should be created with the single line +
 `ATTR{idProduct}=="eb93", ATTR{idVendor}=="10ce", MODE="0666", OWNER="root", GROUP="plugdev"`.
 
 == Standalone Usage
+
 The `xhc-whb04b-6` program can be run from the command line without LinuxCNC to test a pendant.
 This standalone mode is used to identify the button codes produced for each button press and debug transmitted USB data.
 
 === EXAMPLES
 
 `xhc-whb04b-6 -ue`::
-Start in simulation mode and prints incoming USB data transfer and generated key pressed/released events.
+ Start in simulation mode and prints incoming USB data transfer and generated key pressed/released events.
 `xhc-whb04b-6 -p`::
-Start in simulation mode and prints HAL pin names and events distributed to HAL memory.
+ Start in simulation mode and prints HAL pin names and events distributed to HAL memory.
 `xhc-whb04b-6 -H`::
-Start in HAL mode (Normal mode for real machine use).
+ Start in HAL mode (Normal mode for real machine use).
 `xhc-whb04b-6 -HsfB`::
-Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm).
-
+ Start in HAL mode + Spindle Override + Feedrate Override + Big step 5/10&#8239;mm).
```

Comment:
Will the removal of ' +' at the end of the line and before 'Big step', remove a newline?  Is there a '(' missing here?  I only see the ')'.

Reply:
Added the missing '(' , everything else looks as it should look (IMHO), except that you would say "0.5 mm" or "1/2 mm", but not "5/10 mm".